packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -1,37 +1,49 @@
 <template>
-    <!-- NOTE: This is all placeholder markup, attributes and comments.
-    This is not indicative of the actual HTML tags and attributes we
-    will use (which will be much more accessible and semantic) -->
-    <section
-        :class="$style['c-restaurantCard']"
-        data-test-id="restaurantCard-component">
-        <a
-            :href="url"
-            @click="$emit('restaurant-card-clicked')">
-
-            <!-- background image -->
-            <img
-                src=""
-                alt=""
-                loading="lazy"
-                data-test-id="restaurant-cuisine-image">
+    <a
+        :href="url"
+        :class="[
+            $style['c-restaurantCard'],
+            $style[displayModeModifier],
+            { [$style['c-restaurantCard--hasImg']]: !!imgUrl }]"
```

Comment:
This way it will look more consistent and save the need for a computed prop.
```suggestion
            $style['c-restaurantCard'], {
                [$style['c-restaurantCard--listItem']]: isListItem,
                [$style['c-restaurantCard--hasImg']]: !!imgUrl
            }]"
```